public/app/plugins/datasource/prometheus/types.ts

```diff
@@ -63,7 +63,7 @@ export interface PromDataErrorResponse<T = PromData> {
   data: T;
 }
 
-export type PromData = PromMatrixData | PromVectorData | PromScalarData | PromExemplarData[] | null;
```

Comment:
shouldn't we keep `| null` here since previous versions of Prometheus used to return `null`?

Reply:
I don't think we need that. This was only here for exemplars stuff and it is in rc right now.

Reply:
ah sorry, forgot it's not a stable API yet